app/controllers/miq_policy_set_controller.rb

```diff
@@ -0,0 +1,650 @@
+class MiqPolicySetController < ApplicationController
```

Comment:
Why `MiqPolicySet` why not `PolicyProfile`?

Reply:
I am trying to keep all of them to match model name.

Reply:
wdyt @himdel, I'm a little conservative here with the amount of changes, being worried that this might break something,

Reply:
Since I am splitting all the explorers into individual controllers and adding all new views/controllers, i think this change is safe to make. As far as product features are concerned those i would have renamed some anyway, in this PR and in follow up PRs when i start de-explorizing these screens.

Reply:
Well, matching the model name is probably the more conservative solution, the controller name seems to get used as a rbac feature name prefix and as a fallback for table names in GTL and compare.

Then again, deliberately naming it differently would make sure there are no such hidden dependencies, that may be worth it too, and we would end up with more human friendly URLs. No idea if just renaming it would work or if other changes would be needed though.

Reply:
@himdel @skateman are we keeping it as it is now to match models or want to change the names of controller/views/product features prefixes to be more meaningful.

Reply:
Go with models, that's what you've been doing for the previous ones as well, right?

We can always change the URLs on the UI side, having naming consistent with models is probably more ..consistent as far as ruby code goes :)